### ISSUE TITLE
what are self.gain and self.variance in policy_graident_agent.py

### ISSUE THREAD

**Issue by LukasWill** [petroolg/risk-aware-rl#1]
Hi, 
thanks for your great repo. I came across a question about understanding the "reward-update" and "variance-update":
           with tf.name_scope('reward-update'):
                self.reward_loss = tf.losses.mean_squared_error(self.gain, self.sample_reward)
                self.reward_train_op 
                        =tf.train.AdamOptimizer(learning_rate=alpha).minimize(self.reward_loss)
            with tf.name_scope('variance-update'):
                self.variance_loss = tf.losses.mean_squared_error(self.variance, self.sample_variance)
                self.variance_train_op = 
                        tf.train.AdamOptimizer(learning_rate=alpha).minimize(self.variance_loss)
It seems that the update formula of policy do not match the update rule of equation 13 in paper "Policy Gradients with Variance Related Risk Criteria".

- Why sr and cr are not same in policy update, as originally described in paper? it seems both should be R_k?

Thanks a lot in advance for your help!

Best regards
